packages/client/katello-host-tools/katello-host-tools.spec

```
@@ -36,16 +33,16 @@
 %global katello_libdir %{python_libdir}/katello
 
 Name: katello-host-tools
-Version: 4.2.3
-Release: 6%{?dist}
+Version: 4.4.0
+Release: 1%{?dist}
 Summary: A set of commands and yum plugins that support a Katello host
 Group:   Development/Languages
 %if 0%{?suse_version}
 License: LGPL-2.0
 %else
 License: LGPLv2
 %endif
-URL:     https://github.com/Katello/katello-agent
+URL:     https://github.com/Katello/katello-host-tools
 Source0: https://codeload.github.com/Katello/katello-host-tools/tar.gz/%{version}#/%{name}-%{version}.tar.gz
```

I think it's a good practice to use variables.
```suggestion
Source0: https://codeload.github.com/Katello/%{name}/tar.gz/%{version}#/%{name}-%{version}.tar.gz
```